src/axom/slam/examples/HandleMesh.cpp

```diff
@@ -70,7 +70,9 @@ int main(int, char**)
   HandleSet::IndirectionBufferType vecHandle(sz);
 
   // Create a set of handles
-  HandleSet hSet = HandleSet::SetBuilder().size(sz).data(&vecHandle);
+  HandleSet hSet = HandleSet::SetBuilder()  //
+                     .size(sz)              //
+                     .data(&vecHandle);
```

Comment:
I found a trick for enforcing line breaks on chained function calls: Add an empty comment at the end of the line.

Unfortunately, this trick did not work for adding line breaks between parameters of a long function call (as in `axom::determinant()`.